MonoGame.Framework/Graphics/Texture2D.cs

```diff
@@ -93,6 +97,7 @@ protected Texture2D(GraphicsDevice graphicsDevice, int width, int height, bool m
             this.GraphicsDevice = graphicsDevice;
             this.width = width;
             this.height = height;
+            this._texelSize = new Vector2(1f / (float)width, 1f / (float)height);
```

Comment:
> As the code is now you can pass 0 and get a divide by zero in this new code. 

That was true in old code as well. The division moved from SpriteBatch.Draw() into Texture2D().
https://github.com/MonoGame/MonoGame/pull/5267/files#diff-bf9a3678ade2b898dde0ac4d0f4a83cdL391

XNA throws ArgumentOutOfRangeException
`Resource size must be greater than zero.
Parameter name: height`

I guess we need a pattern similar to https://github.com/MonoGame/MonoGame/pull/5121.
to check Texture2D/3D/Cube.